src/packages/solid/components/Column/Column.tsx

```diff
@@ -51,7 +56,10 @@ const Column: Component<ColumnProps> = (props: ColumnProps) => {
       onDown={chainFunctions(props.onDown, onDown)}
       selected={props.selected || 0}
       forwardFocus={onGridFocus}
-      onSelectedChanged={chainFunctions(props.onSelectedChanged, withScrolling(props.y as number))}
+      onSelectedChanged={chainFunctions(
+        props.onSelectedChanged,
+        props.scroll === undefined || props.scroll !== 'never' ? withScrolling(props.y as number) : () => {}
```

Comment:
props.scroll !== 'never' ? withScrolling(props.y as number) : undefined

Reply:
wont this throw an error if props.scroll is undefined?

Reply:
why would it? - chainFunctions was updated to take undefined as arguments